p4c_bm/gen_json.py

```diff
@@ -832,10 +832,17 @@ def dump_actions(json_dict, hlir, p4_v1_1=False):
         for call in action.flat_call_sequence:
             primitive_dict = OrderedDict()
 
-            primitive_name = call[0].name
-            primitive_dict["op"] = primitive_name
-
-            args = call[1]
+            if p4_v1_1 and type(call[0]) is p4.p4_extern_method:
+                # Ref: extern.h, line: 75
```

Comment:
I would remove all the `# Ref` comments here. I think all these points are addressed in the JSON format documentation `https://github.com/p4lang/behavioral-model/blob/master/docs/JSON_format.md`. Maybe you could remove them and instead add a link to the documentation at the top of the file?
These `# Ref` comments are also likely to become invalid pretty fast if we update the bmv2 code.